pegasus/sites.v3/code.org/public/educate/csd.haml

```diff
@@ -230,3 +116,58 @@ theme: responsive_full_width
     = view :support_and_questions_section
 
 = view :analytics_event_log_helper, event_name: AnalyticsConstants::CSD_CURRICULUM_PAGE_VISITED_EVENT
+
+
+%script{:src=>'/js/jquery.carouFredSel-6.2.1-packed.js'}
+%script{:src=>'/js/jquery.touchSwipe.min.js'}
+
+
+:javascript
```

Comment:
@dmantonyuk Javascript files can be added [here](pegasus/sites.v3/code.org/public/js) so they can be linted.